mu-tutoriel.md

```diff
@@ -0,0 +1,199 @@
+# Comment utiliser Mu Editor ?
```

Comment:
Your PR seems to have a lot of unrelated files in the diff, you should probably rebase on master / delete unrelated commits